src/components/main-tools-panel/main-tools-panel.tsx

```diff
@@ -0,0 +1,133 @@
+import { useState } from "react";
+import styled, { css } from "styled-components";
+
+import { darkViolet, primaryViolet } from "../../constants/colors";
+
+import GearIconWhite from "../../../public/icons/gear-white.svg";
```

Comment:
Should we use lazy loading for assets? It flickers on theme switch.

Reply:
I think we can investigate it as separate task.